sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/src/Config/EventHubOptions.cs

```diff
@@ -132,6 +132,27 @@ public int MaxEventBatchSize
             }
         }
 
+        private int _targetUnprocessedEventThreshold;
+
+        /// <summary>
+        /// Get or sets the target unprocessed events per worker for eventhub functions. This is used in target based scaling to override the default behavior of using maxEventBatchSize as the target value.
```

Comment:
nit: `eventhub triggered functions`

Reply:
Event Hubs is the product name; Event Hub is a single resource.    For branding purposes, we shouldn't be using "eventhub".   I believe we'd want to use "Event Hub-triggered functions." in this context.

Reply:
Done, I added Event Hub-triggered functions to description

Reply:
There appears to be build errors now.